### DISCUSSION TITLE
Easily disable a specific task

### DISCUSSION THREAD

**Discussion by elpadi** [phpro/grumphp#902] [Ideas]
| Q               | A
| --------------- | ---
| Version         | 1.3.0
| Bug?            |no
| New feature?    | yes
| Question?       | yes
| Documentation?  | yes
| Related tickets | N.A.

Hi, loving this project so far. I updated the tasks I'm using and one of them seems to have pushed a botched release with a type error in there. I was hoping there would be an easy way to temporarily disable the task, something very simple, clear, and easy to revert, like adding `disabled: true` to the task config.

My "solution" was to comment out the lines for the task in the config. So I'm just asking if there is better way to do it, or requesting the feature if it does not exist.

Thanks

**Comment by janvernieuwe** [comment]
Hi @elpadi ,

You could comment it in the config? I don't think there is anything wrong with that.

**Comment by veewee** [comment]
Commenting it out is an option, but if you are using testsuites, this could result in an issue.
I don't mind adding a property to disable a task through task metadata config. I think it might have its purpose.

**Comment by elpadi** [comment]
> Hi @elpadi ,
> 
> You could comment it in the config? I don't think there is anything wrong with that.

Yes, YAML allows comments by prepending a #.
